src/app/archive/zkapp_tables.sql

```diff
@@ -95,21 +95,30 @@ CREATE TABLE zkapp_verification_keys
 
 CREATE TYPE zkapp_auth_required_type AS ENUM ('none', 'either', 'proof', 'signature', 'both', 'impossible');
 
+CREATE TABLE protocol_versions
```

Comment:
I think this value can remain in `create_schema.sql`

Reply:
since `create_schema.sql` imports `zkapp_tables.sql`, so if I want to refer to this table, I think I have to define it here.

Reply:
It can remain in `create_schema.sql`, but it needs to be before the import of `zkapp_tables.sql`.

I'd like `zkapp_tables.sql` to contain only tables that are specific to zkApps.